modules/scheduler/gke-cluster/main.tf

```diff
@@ -37,6 +37,23 @@ locals {
 
   # multi networking needs enabled Dataplane v2
   derived_enable_dataplane_v2 = coalesce(var.enable_dataplane_v2, local.derived_enable_multi_networking)
+
+  default_monitoring_component = [
```

Comment:
Are these all free by default ?

Reply:
there is always cost for storage, but these are the minimal observability a cluster should have. And it is the default previously when the toolkit is using monitoring_service and logging_service as well.

Reply:
free for GKE Enterprise https://cloud.google.com/stackdriver/pricing?_gl=1*10s754z*_ga*MTE3Mjk1NjYyMy4xNzMxOTU3NDc1*_ga_WH2QY8WWF5*MTczMTk2MzcwNS4yLjEuMTczMTk2Nzk3Ny40Ny4wLjA.#anthos-costs